Containerfile

```diff
@@ -28,18 +23,18 @@ COPY deploy/ deploy/
 COPY hack/ hack/
 COPY internal/ internal/
 
-RUN make build
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -mod=vendor --ldflags "-s -w" -a -o bin/kubesan cmd/main.go
```

Comment:
Yes that was the idea, I pinged you in slack to discuss CI a little more. I think I need to add some context around what you have for CI

Reply:
Since `make build` is no longer part of Containerfile, I am curious how we will call it in CI. I think the CI env should be updated to accommodate these changes before they get merged; otherwise, it will be broken again.

Reply:
The simplest fix to keep CI happy before we go with this full PR:
```
printf 'container-build:\n\tpodman build -t kubesan:latest .\n' >> Makefile
```
and update CI to call 'make container-build' where it now calls './containerized-build.sh'. At that point, containerized-build.sh becomes unused and can be removed again.

Tweak the Makefile line to use proper overrides (for example, we should reinstate the `CONTAINER_TOOL ?= 
podman` line that was lost in commit 37e428, and call $(CONTAINER_TOOL) instead of bare podman)

Reply:
CI call to build is a one liner change. It´s easy enough to me to patch at any given time. Just use the approach you guys prefer and I will just CI.